test/fixtures/output/schema-registry/config/describe-subject.golden

```diff
@@ -1,21 +1,21 @@
-+---------------------+----------------------------------------+
-| Compatibility Level | FORWARD                                |
-| Compatibility Group | application.version                    |
-| Metadata Defaults   | {                                      |
-|                     |   "properties": {                      |
-|                     |     "email": "bob@acme.com",           |
-|                     |     "owner": "Bob Jones"               |
-|                     |   }                                    |
-|                     | }                                      |
-| Ruleset Defaults    | {                                      |
-|                     |   "domainRules": [                     |
-|                     |     {                                  |
-|                     |       "name": "checkSsnLen",           |
-|                     |       "kind": "CONDITION",             |
-|                     |       "mode": "WRITE",                 |
-|                     |       "type": "CEL",                   |
-|                     |       "expr": "size(message.ssn) == 9" |
-|                     |     }                                  |
-|                     |   ]                                    |
-|                     | }                                      |
-+---------------------+----------------------------------------+
++---------------------+-----------------------------------------+
+| Compatibility Level | FORWARD                                 |
+| Compatibility Group | application.version                     |
+| Metadata Defaults   | {                                       |
+|                     |   "properties": {                       |
+|                     |     "email": "bob@acme.com",            |
+|                     |     "owner": "Bob Jones"                |
+|                     |   }                                     |
+|                     | }                                       |
+| Ruleset Defaults    | {                                       |
+|                     |   "domainRules": [                      |
+|                     |     {                                   |
+|                     |       "expr": "size(message.ssn) == 9", |
```

Comment:
Hopefully the order won't change every time we run these tests?

Reply:
no. It's alphabetic ;)

Reply:
you can see semaphore tests pass every time